lib/v8debugapi.js

```diff
@@ -220,6 +220,22 @@ module.exports.create = function(logger_, config_, fileStats_) {
    * @param {function(?Error)} cb error-back style callback
    */
   function setInternal(breakpoint, scriptPath, compile, cb) {
+    // The breakpoint protobuf message presently doesn't have a column property
+    // but it may have one in the future.
+    var column = breakpoint.location.column || 1;
+    var line = breakpoint.location.line;
+
+    // We need to special case breakpoints on the first line. Since Node.js
+    // wraps modules with a function expression, we adjust
+    // to deal with that.
+    if (semver.satisfies(process.version, '>=5.2')) {
```

Comment:
I don't think this is necessary just yet given that the breaking change was reverted in 5.3.